SU-HSDO/suhumsci
# automatically set importer permissions when configured

# READY FOR REVIEW

# Summary
- Automatically allow site managers to run importers _after_ importer has been configured.

# Needed By (Date)
- asap

# Urgency
- medium

# Steps to Test
1. checkout this branch
1. composer install
1. blt drupal:sync
1. go to permission page, verify site managers don't have permission to "Execute Migration Course importer" or "Execute Migration Events importer"
1. configure an events or course migration at `/admin/structure/migrate/course-importer` or `/admin/structure/migrate/events`
1. go back to permission page, verify site managers now have permission to "Execute Migration Course importer" or "Execute Migration Events importer".



# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
